src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext, useMemo } from 'react';
+import classNames from 'classnames';
+import { TodoContext } from '../Main/TodosContext';
+import { ListAction } from '../Enum/ListAction';
+import { Todos } from '../Types';
+
+export const Footer: React.FC = () => {
+  const {
+    todo,
+    setTodo,
+    filter,
+    setFilter,
+  } = useContext(TodoContext);
+
+  const selectedTodos = useMemo(() => (
```

Comment:
there is only todos length, so make sense to rename `selectedTodos` to `selectedTodosLength` or `selectedTodosCount`